src/guides/v2.2/ui_comp_guide/components/ui-textarea.md

```diff
@@ -7,92 +7,44 @@ The Textarea component implements the `<textarea>` form field.
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>component</code>
-    </td>
-    <td>
-      The path to the component’s .js file in terms of RequireJS.
-    </td>
-    <td>
-      String
-    </td>
-    <td>
-      <code>'Magento_Ui/js/form/element/textarea'</code>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>elementTmpl</code>
-    </td>
-    <td>
-      The path to the <code>.html</code> template of the particular
-      type of field (textarea).
-    </td>
-    <td>
-      String
-    </td>
-    <td>
-      <code>'ui/form/element/textarea'</code>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>rows</code>
-    </td>
-    <td>
-      The number of rows that will be specified in the
-      <code>rows</code> attribute of the textarea DOM element.
-    </td>
-    <td>
-      Number
-    </td>
-    <td>
-      <code>2</code>
-    </td>
-  </tr>
-  <tr>
-    <td>
-      <code>template</code>
-    </td>
-    <td>
-      The path to the general field <code>.html</code> template.
-    </td>
-    <td>
-      String
-    </td>
-    <td>
-      <code>'ui/form/field'</code>
-    </td>
-  </tr>
-</table>
+| Option | Description | Type | Default |
+| --- | --- | --- | --- |
+| `cols` | The number of columns that will be specified in the `cols` attribute of the textarea DOM element. | Number | `15` |
+| `component` | The path to the component’s `.js` file in terms of RequireJS. | String | `'Magento_Ui/js/form/element/textarea'` |
+| `elementTmpl` | The path to the `.html` template of the particular type of field (textarea). | String | `'ui/form/element/textarea'` |
+| `rows` | The number of rows that will be specified in the `rows` attribute of the textarea DOM element. | Number | `2` |
+| `template` | The path to the general field `.html` template. | String | `'ui/form/field'` |
+
+## Source files
+
+Extends [`Abstract`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/form/element/abstract.js):
+
+-  [app/code/Magento/Ui/view/base/web/js/form/element/textarea.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/form/element/textarea.js)
+-  [app/code/Magento/Ui/view/base/web/templates/form/element/textarea.html]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/form/element/textarea.html)
+-  [app/code/Magento/Ui/view/base/web/templates/form/field.html]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/form/field.html)
+
+## Examples
+
+### Integrate the Textarea component with the Form component
+
+This example integrates the Textarea component with the [Form]({{ page.baseurl }}/ui_comp_guide/components/ui-form.html) component
+
+```xml
+<form>
+    ...
+    <fieldset>
+        ...
+        <textarea name="textarea_example">
+            <settings>
+                <cols>15</cols>
+                <rows>5</rows>
+                <label translate="true">Textarea Field Example</label>
```

Comment:
Hi @serhiyzhovnir. Thank you for the updated information. Maybe it makes sense to add a description of the "label" field to the table above as well? (as we do in https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-text.html, for example). What do you think?